internal/commands/user.go

```diff
@@ -35,3 +37,21 @@ func getSlackUserInfo(
 
 	return &user, err
 }
+
+func getUsersInConversationParameters(
+	channelID string,
+	cursor string,
+) *slack.GetUsersInConversationParameters {
+
+	//Check if a cursor for a next page was received
+	if cursor != "" {
```

Comment:
:thinking: This was resolved here:
https://github.com/ResultadosDigitais/hellper/pull/80/files#diff-48f4e046d73ead02d5aeecd418f846e0R71

Reply:
I was planning to do this on another function, which only creates the struct, but I liked your solution.
I think we could close this PR and use only yours.

What do you think @tiagoemsi?

Reply:
I agree with you